projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/my-company/my-company-list.ts

```diff
@@ -0,0 +1,80 @@
+import { MyCompanyConfig } from './models/index';
+import {
+  loginAsMyCompanyAdmin,
+  waitForData,
+  verifyList,
+  getListRowsFromBody,
+} from './my-company.utils';
+
+const DEFAULT_SORT_LABEL = 'name';
+
+export function testListFromConfig(config: MyCompanyConfig): void {
+  describe(`${config.name} List`, () => {
+    beforeEach(() => {
+      loginAsMyCompanyAdmin();
+      cy.server();
+    });
+
+    it('should show and paginate list', () => {
+      cy.visit(`/organization`);
+      testList(
+        config,
+        cy.get(`cx-page-slot.BodyContent a`).contains(config.name).click()
+      );
+    });
+
+    testListSorting(config);
+  });
+}
+
+export function testList(
+  config: MyCompanyConfig,
+  trigger: any,
+  callback?: Function
+): void {
+  cy.route('GET', `**${config.apiEndpoint}**`).as('getData');
+  waitForData((data) => {
+    const listData = getListRowsFromBody(data, config.objectType, config.rows);
+    verifyList(listData, config.rows);
+
+    if (data.pagination.currentPage < data.pagination.totalPages - 1) {
+      testPagination(data);
+    }
+
+    if (callback) {
+      callback(listData);
+    }
+  }, trigger);
+
+  function testPagination(data: any) {
+    testList(
+      config,
+      cy
+        .get(`cx-pagination a.page`)
+        .contains(data.pagination.currentPage + 2)
+        .click()
+    );
+  }
+}
```

Comment:
As we already spoke about this, let's not test all pages and pagination. I believe that one or two clicks would be more than sufficient. We came to the point where test takes ages testing 30+ pages